druid-handler/src/test/org/apache/hadoop/hive/druid/QTestDruidSerDe.java

```diff
@@ -27,12 +27,16 @@
 
 import org.apache.druid.query.metadata.metadata.SegmentAnalysis;
 import org.apache.druid.query.metadata.metadata.SegmentMetadataQuery;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Druid SerDe to be used in tests.
  */
 public class QTestDruidSerDe extends DruidSerDe {
 
+  public static final Logger log = LoggerFactory.getLogger(QTestDruidSerDe.class.getName());
```

Comment:
There are no log messages in this class (and in some other SerDe classes) so there is no reason to introduce unused logger variables in those.

Reply:
Again I am not worried about the log in this file but rather in the parent class that want to log the right instantiated class name. At https://github.com/apache/hive/blob/cbd6363ce904ea9150a8e8591d60edfc6b4adea9/serde/src/java/org/apache/hadoop/hive/serde2/AbstractSerDe.java#L93